.gitmodules

```diff
@@ -18,3 +18,6 @@
 	path = library/parsedown
 	url = https://github.com/mantisbt/parsedown.git
 	branch = mantisbt
+[submodule "api/rest/swagger"]
```

Comment:
There is two components to swagger:

- **Swagger.json** - It is the meta-data describing the APIs exposed by the RESTful service (equivalent of WSDL for SOAP).  This file is used to power Swagger UI or code generators that generate clients in multiple languages based on Swagger meta-data.
- **Swagger UI** - This is a JS web app that reads such swagger.json and provides the page that describes all APIs including documentation, authentication support, etc.  API consumers can use this UI to understand the API as well as try it out.  Think of it like a SoapUI or one of these tools.  Bundling this enables that for any MantisBT instance.